src/installer.js

```diff
@@ -11,6 +11,7 @@ var glob = require('glob')
 var path = require('path')
 var temp = require('temp').track()
 var wrap = require('word-wrap')
+var mkdirp = require('mkdirp')
```

Comment:
`fs-extra` already has this functionality (https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir.md), so no need to add more dependencies.

Reply:
Appears to be unofficially supported: https://github.com/jprichardson/node-fs-extra/issues/60

Reply:
I guess when it becomes officially supported we can switch to using only `fs-extra` in the meantime we should stick with `mkdirp`